wasp_pickpocketer.simba

```diff
@@ -771,7 +771,7 @@ begin
       OPEN_BANK: Bank.WalkOpen();
       DEPOSIT_LOOT: Self.Deposit();
       WITHDRAW_FOOD: Bank.WithdrawConsumable(ERSConsumable.FOOD);
-      WITHDRAW_NECKLACES: Self.Withdraw([Necklace, NeckAmount, False]);
+      WITHDRAW_NECKLACES: Bank.WithdrawItem([Necklace, NeckAmount, False], False);
```

Comment:
Also updated the bank withdraw call.

If you have "Use X" selected in bank, withdrawing of necklaces will withdraw that many. 
For example: if the bot sets X to 10 for food, then it will end up withdrawing 10 necklaces next time it banks.

Updated to use the Bank.WithdrawItem with `useQuantityButtons = False`